content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -97,6 +153,8 @@ To encrypt the data:
 
 1. Create a new encryption configuration file using the appropriate properties for the `kms` provider:
 
+{{< tabs name="tab_with_md" >}}
+{{% tab name="v1" %}}
```

Comment:
Could we list the supported resources? 
I had success with ConfigMap as an example.